src/lib/sidebar/PipelineSchemeForm.svelte

```diff
@@ -10,15 +10,18 @@
     TextArea,
     TextInput,
   } from "lib/govuk";
-  import { gjScheme } from "stores";
+  import { gjSchemeCollection } from "stores";
   import ATF4Type from "../forms/ATF4Type.svelte";
+  import { getFirstSchemeOrEmptyScheme } from "./scheme_data";
 
   let showModal = false;
+  let scheme = getFirstSchemeOrEmptyScheme($gjSchemeCollection);
+  $gjSchemeCollection.schemes[scheme.scheme_reference] = scheme;
 
   // This component is only created once, but gjScheme could be cleared
   // completely multipel times. Set defaults anytime the modal is open.
   $: if (showModal) {
-    $gjScheme.pipeline ||= {
+    scheme.pipeline ||= {
```

Comment:
I can check when I'm home, but can you make sure the indirection works with Svelte? Every time you edit a pipeline form field, in the console you should see it saving the GJ to local storage. This was a subtle bug a few PRs back

EDIT: never mind, you're copying things over in `onCompletion`. That should be fine. Maybe a comment there to note nothing is saved until then, because it's unfortunate Svelte subtlety